bin/setup

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Similar gripes as `bin/console`, but it is made worse as this only works for `*nix` users, and is completely un-usable for `Windows` users.